ci/pipeline-vps.yml

```diff
@@ -0,0 +1,81 @@
+resources:
```

Comment:
Seems this is caused by PR dependency, it is supposed to be only 2 files in commit 	[310fb54](https://github.com/cloudfoundry-community/bosh-softlayer-baremetal-server-release/pull/27/commits/310fb54ee58e269bc10eb6c898fa8cfc241674b8).